src/handlers.js

```diff
@@ -10,6 +10,33 @@ const {consume} = require('taskcluster-lib-pulse');
 const debugPrefix = 'taskcluster-github:handlers';
 const debug = Debug(debugPrefix);
 
+const TITLES = { // maps github checkruns statuses and conclusions to titles to be displayed
+  success: 'Success',
+  failure: 'Failure',
+  neutral: 'It is neither good nor bad',
+  cancelled: 'Cancelled',
+  timed_out: 'Timed out',
+  action_required: 'Action required',
+  queued: 'Queued',
+  in_progress: 'In progress',
+  completed: 'Completed',
+};
+
+const CONCLUSIONS = { // maps queue exchange status to github checkrun conclusion
```

Comment:
I guess most of these aren't really exchange statuses but rather `reason`s for exception tasks. `completed`, `failed`, `exception` are queue statuses but the rest aren't.

Reply:
yeah, forgot to edit the comment